### DISCUSSION TITLE
How do you disable background darkening?

### DISCUSSION THREAD

**Discussion by JohannD123** [Godiesc/firefox-one#26] [Q&A] [answered]
I've tried modifying the root personal css file but I'm not seeing any changes to my browser.

I already have the main image enabled, is there something else I need to enable within the browser? I thought you could configure things through the css.

![image](https://github.com/Godiesc/firefox-one/assets/156850013/6c1b2ce2-5101-4210-8d41-df6755225c3b)

**Comment by Godiesc** [comment, marked as answer]
hi, for brightness you have the rules in line 60 and 62, in the section for main-image config: https://github.com/Godiesc/firefox-one/blob/main/chrome/components/one_root-personal.css#L60-L62 , you can edit the last 2 numbers and write something like this: #10101900 for transparency.
